src/System.IO.FileSystem/src/System/IO/DirectoryInfo.cs

```diff
@@ -2,8 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
Here and below, these usings were unused.

Reply:
They're used by the redist build.